commit a362006e2748e5d62253fa5d53c51d213c68b4eb (refs/changes/37/5337/2)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2019-01-10T10:50:11-08:00 (9 months ago)
    
    TRITON-1075 mockcloud CNs should have Admin IP in sysinfo
    Reviewed by: Trent Mick <trentm@gmail.com>
    Reviewed by: Rui <rui@joyent.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/lib/backends/dummy/tools/create-server.js b/lib/backends/dummy/tools/create-server.js
index b5abeca..1614b88 100644
--- a/lib/backends/dummy/tools/create-server.js
+++ b/lib/backends/dummy/tools/create-server.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2018, Joyent, Inc.
+ * Copyright (c) 2019, Joyent, Inc.
  */
 
 //
@@ -172,6 +172,10 @@ function populateNics(ctx, callback) {
                     assert.string(provRes.ip, 'res.ip');
                     nics[nicName].ip4addr = provRes.ip;
 
+                    // To match platforms containing TRITON-841, we want to also
+                    // set 'Admin IP' at the top level in sysinfo.
+                    ctx.serverObj['Admin IP'] = provRes.ip;
+
                     cb();
                 });
             } else {
diff --git a/package.json b/package.json
index a7fd8d5..20684fd 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cn-agent",
   "description": "Triton Compute Node Agent",
-  "version": "2.6.2",
+  "version": "2.7.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
